shunting-yard-replicator/src/main/java/com/hotels/shunting/yard/replicator/exec/conf/ct/SyReplicaTable.java

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (C) 2016-2019 Expedia Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.hotels.shunting.yard.replicator.exec.conf.ct;
+
+import org.hibernate.validator.constraints.NotBlank;
+
+public class SyReplicaTable {
```

Comment:
I prefer `ReplicaTable`

Reply:
This is SyReplicaTable because the CT's ReplicaTable is being used in the `CircusTrainConfig` class in SY. So, if I name it same as CT, it creates really ugly scenes in that class.

Reply:
This class is removed as I am reusing CT one's now.

Reply:
ok so I know we started out with reusing this but then the validation became an issue. I'm not sure we should use it now. Consider the following in CT we add a validation to one of the ReplicaSource fields now we bump the version in SY to use that. Everything compiles just fine. But we'll get runtime exception when running stuff. Not sure we if we will catch all of that in tests. thoughts?

Reply:
Hmm.. good point.. I am only using dbName and tableName fields here. So, I feel like those two fields will perhaps not get any extra validation checks on them coz they are optional in CT.

Its just that if we copy those classes, it all looks a little ugly coz I need to use CT's ReplicaTable and SourceTable classes to build CT Config (In ContextFactory). It then causes confusion in the project to have both of them. But I will go with what everyone agrees on.

Reply:
doesn't matter what you use. if a field gets added with @notNull validation it will fail. I know this would mean a major CT version and we should notice, nonetheless everything will compile it will either be caught in a test (best case) or at runtime (worst case). We do have this problem when building the CT config as well though so maybe it's ok. Dunno :(